spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+set -e  # abort on error
+
+if [ -z "$API_HOST" ]; then
+  echo "API_HOST unset, trying minishift"
+  if which minishift && minishift status | grep -i 'openshift:.*running'; then
+    export API_HOST="$(minishift ip)"
+    eval $(minishift oc-env --shell bash) # Ensure oc in PATH
+    oc login -u system:admin # With minishift, we know we can just do this
+  else
+    echo 'Either set $API_HOST and perform `oc login` with admin powers,'
+    echo 'or have minishift in $PATH and already running, e.g.'
+    echo '    minishift addons enable manageiq'
+    echo '    minishift start --vm-driver virtualbox --openshift-version v3.7.0'
+    exit 1
```

Comment:
you could add a link to this page explaining deployment options:
http://manageiq.org/docs/guides/providers/openshift